src/main/kotlin/com/pubnub/api/managers/RetrofitManager.kt

```diff
@@ -25,13 +25,13 @@ import java.util.concurrent.TimeUnit
 
 class RetrofitManager(val pubnub: PubNub) {
 
-    private lateinit var transactionClientInstance: OkHttpClient
+    private lateinit var transactionClientInstance: OkHttpClient // TODO it doesn't seem right that these are lateinit
```

Comment:
What we wan't to do about this. If don't know how to handle this right now maybe Jira ticket would be better not to forget about this?

Reply:
ok, removing for now